ansible/README.md

```diff
@@ -113,7 +121,7 @@ export OW_DB_PROTOCOL=https
 export OW_DB_HOST=<your cloudant user>.cloudant.com
 export OW_DB_PORT=443
 
-ansible-playbook -i environments/<environment> setup.yml
+ansible-playbook -i environments/<environment> couchdb.yml --tags ini
```

Comment:
This `--tags` is new to me.
It will only perform the tasks tag with `ini` and ignore all the rest in the playbook couchdb.yml ?